src/main/kotlin/com/openlattice/chronicle/study/StudySettings.kt

```diff
@@ -0,0 +1,21 @@
+package com.openlattice.chronicle.study
+
+import com.openlattice.chronicle.sensorkit.SensorType
+
+/**
+ * @author alfoncenzioka &lt;alfonce@openlattice.com&gt;
+ */
+
+class StudySettings(val settings: Map<String, Any>) {
```

Comment:
This seems like it should be in study object? You could move the code here into the study object with minimal changes.

Reply:
Moved